buildrunner/__init__.py

```diff
@@ -250,6 +251,40 @@ def _re_split_filter(text, pattern, maxsplit=0, flags=0):
         '''
         return re.split(pattern, text, maxsplit=maxsplit, flags=flags)
 
+    @staticmethod
+    def _reorder_dependency_steps(config):
+        """
+        Reorders the steps based on the dependencies that are outlined in the config
+        """
+        # Defines configuration keywords, should add to a config validation class
+        keyword_version = 'version'
+        keyword_steps = 'steps'
+        keyword_depends =   'depends'
+        supported_version = 2.0
```

Comment:
I choose version 2.0 arbitrary. It may be easier to remind and maintain future use of this version number in the configuration to match the version of `buildrunner`. Thoughts on this?

Reply:
Yes, I agree.

Reply:
Ok I will switch to be the first 2 numbers of the current version (MAJOR:MINOR) so that I can easily do a numerical comparison. I will bump up the minor number of the version.

Reply:
Nevermind, after talking with you I will bump the buildrunner version to 2.0